src/main/java/org/jenkinsci/plugins/configfiles/GlobalConfigFiles.java

```diff
@@ -52,6 +57,11 @@ public static GlobalConfigFiles get() {
         return instance;
     }
 
+    @Override
+    public String getGlobalConfigPage() {
+        return null;
+    }
```

Comment:
@ndeloof so this is actually the only change that is need to support CasC even better?
...given `Config` is abstract because of #54 already